doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,148 @@
 
 <a name="2.3.1"></a>
 ### 2.3.1 General
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.gen.ost.01` | Open source | The Architecture **must** use Kubernetes APIs.|
-| `req.gen.ost.02` | Open source | The Architecture **must** support dynamic request and configuration of virtual resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.k8s.01` | Open source | The Architecture **must** use Kubernetes APIs.|
+| `req.gen.k8s.02` | Open source | The Architecture **must** support dynamic request and configuration of resources (compute, network, storage) through Kubernetes APIs. |
+| `req.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. |
+| `req.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. |
+| `req.gen.cnt.03` | Cloud nativeness | The Architecture **must** support immutable infrastructure. |
+| `req.gen.scl.01` | Scalability | The Architecture **should** support policy driven horizontal auto-scaling of workloads. |
+| `req.gen.rsl.01` | Resiliency | The Architecture **must** support resilient Kubernetes components that are required for the continued availability of running workloads. |
+| `req.gen.rsl.02` | Resiliency | The Architecture **should** support resilient Kubernetes service components that are not subject to `req.gen.rsl.01`. |
+| `req.gen.avl.01` | Availability | The Architecture **must** provide High Availability for Kubernetes components. |
+| `req.gen.ref.01` | Model | The Architecture **must** support the Reference Model defined profiles (instance types: Basic, Network Intensive, Compute Intensive). |
+
+
+<p align="center"><b>Table 2-1:</b> Kubernetes Architecture: General Requirements</p>
+
+<a name="2.3.2"></a>
+### 2.3.2 Infrastructure
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.inf.com.01` | Compute | The Architecture **must** provide compute resources for Pods.  |
+| `req.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). |
+| `req.inf.com.03` | Network | The Architecture **must** support Conatiner Runtime Interface (CRI). |
+| `req.inf.stg.01` | Storage | The Architecture **must**  support for Kubernetes Volumes for container storage.
```

Comment:
| `req.inf.stg.01` | Storage | The Architecture **must**  support for Kubernetes persistent Volumes for container storage
@pgoyal01 as agreed

Reply:
Persistent volumes are a type of Kubernetes volume. We need to include non persistent volumes as well as persistent volumes.

Reply:
E.g. I could choose to use a Cinder volume direct from the container spec, rather than using a PV+PVC. https://kubernetes.io/docs/concepts/storage/volumes/#cinder

Reply:
@tomkivlin , You are right, the issue here as per our last Friday's call, all the volume listed in the mentioned link consider, the backend storage that can be supported ( as per my understanding )
So my POV to remove this clause `req.inf.stg.01` as the backend storage should be the same either for the container workload or underneath runtime 
what did you think @tomkivlin

Reply:
@ASawwaf I don't think the storage needs to be the same as that used by the runtime, no. I don't think specifying Persistent Volumes instead would do that either. Perhaps we need two different requirements, such as:
1. The Architecture must support ephemeral storage (non-persistent) storage for Pods.
2. The Architecture must support persistent storage for Pods.

I've specifically referred to Pods and not containers as I'm not sure its possible to provide storage that follows the lifetime of a container only, not the pod.

Reply:
@tomkivlin , I am ok with the 2 above points 
@pgoyal01 can you please add them 

Thanks

Reply:
| `req.inf.stg.06` | Storage | The Architecture **must** support ephemeral storage (non-persistent) storage for Pods. |
| `req.inf.stg.07` | Storage | The Architecture **must** support persistent storage for Pods. |